integration/spark/src/main/scala/org/apache/spark/sql/execution/command/view/CarbonCreateMVCommand.scala

```diff
@@ -156,10 +157,18 @@ case class CarbonCreateMVCommand(
 
   private def doCreate(session: SparkSession,
       tableIdentifier: TableIdentifier,
-      viewManager: MVManagerInSpark): MVSchema = {
+      viewManager: MVManagerInSpark,
+      viewCatalog: MVCatalogInSpark): MVSchema = {
     val logicalPlan = MVHelper.dropDummyFunction(
       MVQueryParser.getQueryPlan(queryString, session))
-    val modularPlan = checkQuery(session, logicalPlan)
+      // check if mv with same query already exists
+    val mvSchemaWrapper = viewCatalog.getMVWithSameQueryPresent(logicalPlan)
+    if (mvSchemaWrapper.nonEmpty) {
+      throw new MalformedMVCommandException(
+        "MV with name `" + mvSchemaWrapper.get.viewSchema.getIdentifier.getTableName +
+        "` has been already created with same query")
```

Comment:
1. do not use `+`, please use like scala srting concats `s"xxxxx ${dddd} xxxxx"` 
2. can change to `MV with the name has been already created with the same query`

Reply:
done